### ISSUE TITLE
a user should be able to quickly filter between online and in-person sessions

### ISSUE THREAD

**Issue by csell5** [ThatConference/that.us#779]
**Is your feature request related to a problem? Please describe.**
right now there is no way to filter between online and in-person sessions.

**Describe the solution you'd like**
* at a min we should have filter criteria in the search box. That search criteria is persistent between sessions. so it's not a bad solution.
* adding buttons to toggle search feels weird and like more clutter on the screen.
* open to other ideas

**Comment by SQrL143** [comment]
When activities are created, was there a differentiator at creation?  Just trying to tie making an activity (open space etc. ) to filtering them apart, but I am not sure of what is collected. I added an ON THAT tag to my activity, but I imagine you don’t want to rely on people’s memory to tag nor to burden creators with a specific tag

**Comment by csell5** [comment]
Yes there is a type. If you go to api.that.tech there is a docs. Search for the type session and you'll find all the fields.

My thinking was we'd just add that type to the search fly out as another category. Those search selections are persisted in session storage as well.
